pubsub/client.go

```diff
@@ -122,11 +124,12 @@ const (
 )
 
 // Creates a new Radix client and subscribes to it.
-func New(address string) *Client {
+func New(address, password string) *Client {
```

Comment:
I had a similar problem the other night!  It is frustrating indeed.  Thanks in advance!